aml_test.go

```diff
@@ -0,0 +1,77 @@
+package yoti
+
+import (
+	"io/ioutil"
+	"net/http"
+	"strings"
+	"testing"
+
+	"gotest.tools/assert"
+	is "gotest.tools/assert/cmp"
+)
+
+func createStandardAmlProfile() (result AmlProfile) {
+	var amlAddress = AmlAddress{
+		Country: "GBR"}
+
+	var amlProfile = AmlProfile{
+		GivenNames: "Edward Richard George",
+		FamilyName: "Heath",
+		Address:    amlAddress}
+
+	return amlProfile
+}
+func TestYotiClient_PerformAmlCheck_Success(t *testing.T) {
```

Comment:
I know you've just moved these files, and not added them, but it would be good to be consistent and have a linebreak in between functions (also applies to other parts of this PR too)

I just looked up whether `gofmt` should be doing this, but they made a decision not to autoformat this based on function groups like this:
```go
func (n *Node) Noescape() bool { return n.flags&nodeNoescape != 0 }
func (n *Node) Bounded() bool  { return n.flags&nodeBounded != 0 }
```